plugins/nominal-connection-checker/src/type_hierarchy.js

```diff
@@ -497,16 +497,17 @@ export class TypeHierarchy {
    * parameters of the actualTypes that are associated with the given parameter
    * at that index for the common type.
    * @param {!Array<!TypeStructure>} actualTypes The actual types we are mapping
-   *     the paramters of to the common type.
+   * the paramters of to the common type.
    * @param {string} commonType The name of the common type we are getting the
-   *     params in the order of.
+   * params in the order of.
+   * @param getParamsForCommon
    * @param {function(!TypeDef, string, !Array<!TypeStructure>)
-   * :!Array<!TypeStructure>}
-   *     getParamsForCommon A function that takes in the definition of a type
-   *     and maps the given actual params to the parameters of the common type.
+:!Array<!TypeStructure>}
```

Comment:
Could you revert all the changes that aren't in the backpack plugin? They aren't related and some of the comments are incorrectly formatted - e.g. not starting the comment line with the asterisk can cause problems with jsdoc parsing.

Reply:
Yeah I was wondering about all those changes. I couldn't get the project to run due to linting errors. And I ran the lint:fix command, which changed a lot of files.